lib/custom-logger.js

```diff
@@ -83,7 +83,7 @@ function logRequestCompleted(req, reply, next) {
       response: {
         statusCode,
         body: {
-          bytes: reply.getHeader('content-length'),
+          bytes: parseInt(reply.getHeader('content-length')) || undefined,
```

Comment:
undefined or 0?

```suggestion
          bytes: parseInt(reply.getHeader('content-length'), 10) || undefined,
```

or Number(reply.getHeader('content-length'))

Reply:
`undefined`, there might still be a content; better not telling anything than telling that there's empty content.